Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml

```
@@ -229,8 +229,8 @@
 - type: entity
   parent: ClothingNeckBase
   id: ClothingNeckCloakBi
-  name: poison cloak
-  description: The purple color is a clear indicator you are poisonous.
+  name: biathalon cloak
+  description: A cloak for folk in multiple competitions
```

grammar nitpick (godo)
```suggestion
  description: A cloak for folk in multiple competitions.
```